src/act.core.web/Services/INotifier.cs

```diff
@@ -227,12 +227,14 @@ private async Task NotifyComplianceByChefId(AutomateWebHookMessage message)
                     AddToLists(bs.Parent?.Owner, LevelConstants.Boss, ref emails, ref names);
                 }
                 
-                if (result.FailingSince < now.AddDays(-60))
-                {
-                    AddToLists(result.Owner, LevelConstants.BossesBoss, ref emails, ref names);
-                    AddToLists(bs.Owner, LevelConstants.BossesBoss, ref emails, ref names);
-                    AddToLists(bs.Parent?.Owner, LevelConstants.BossesBoss, ref emails, ref names);                   
-                }
+                // Escalate mails to BossesBoss
```

Comment:
I thought you were going to keep the 60 days, just remove the part that adds the third level of bosses.